app/containers/Quiz/index.js

```diff
@@ -0,0 +1,36 @@
+import React, {Component} from 'react';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+
+import * as quizActionCreators from 'actions/quiz';
+
+const assembledActionCreators = Object.assign({}, quizActionCreators);
+
+import Question from 'components/Question';
+
+import {styles} from './styles.scss';
+
+const mapStateToProps = state => {
+  return {
+    question: state.quiz.question
+  };
+}
+
+@connect (
```

Comment:
The `@connect` syntax is being enabled by `babel-plugin-transform-decorators-legacy`, but technically decorators will be part of ES7, so is not standard yet, right? Wonder if we should stick with ES6 syntax until we officially adopt ES7 when it's ready. Just for simplicity so people don't need to look up what exactly this is. Just a thought.